impeller/docs/android_cpu_profile.md

```diff
@@ -0,0 +1,55 @@
+
+# Android CPU Profiling
+
+Android devices have different performance characteristics than iOS devices, and I've found that CPU traces frequently reveal surprising performance issues, such as https://github.com/flutter/engine/pull/48303 . This document describes the steps to capture an equivalent [flame graph](https://cacm.acm.org/magazines/2016/6/202665-the-flame-graph/abstract) on your local Android device.
```

Comment:
Scratch "I've found that", as it's reviewed/in our repo it doesn't need an "I found".

Reply:
Done